src/components/common/YourCompany/BusinessType.vue

```diff
@@ -2,11 +2,12 @@
   <div
     id="business-type"
     class="section-container"
-    :class="{'invalid-section': invalidSection}"
+    :class="{'invalid-section': invalidSection || isShowingError}"
   >
```

Comment:
use the invalid-section style for this section before user click "Review and Certify"